src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_FIGURES = 5;
+    private static final int LENGTH_MAX = 100;
+    private static final int CIRCLE_DEFAULT_RADIUS = 10;
+    private static final double CIRCLE_DEFAULT_AREA = 314.0;
+
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int choice = random.nextInt(NUMBER_FIGURES);
+        switch (choice) {
+            case 1:
+                Circle circle = new Circle();
+                circle.setColor(color.getRandomColor());
+                circle.setRadius(random.nextInt(LENGTH_MAX));
+
+                return circle;
```

Comment:
```suggestion
            case 0:
                return getRandomCircle();
```
lets add private methods to get each random figure - this would make your code easier to read